### ISSUE TITLE
Won't compile after updating command-line tools to version 15

### ISSUE THREAD

**Issue by Naktakala** [iains/gcc-12-branch#22]
My M2 macbook auto updated to command-line tools version 15 and now I can link programs i used to be able to. And I cannot compile gcc-13. Neither does homebrew

**Comment by iains** [comment]
yeah, there is a problem with the current Xcode 15 command line tools - we believe that it will be fixed in a future  dot release;  in ht meantime, probably the best solution is to re-install 14.3 (you should be able to download it from the developer.apple.com website, although you do need an apple ID to do that)

**Comment by iains** [comment]
As far as I know, with the current branch (https://github.com/iains/gcc-12-branch/commit/99533d94172ed7a24c0e54c4ea97e6ae2260409e) for Xcode 16 (SDK MacOS15.sdk) should be working - please let me know if issues remain.
